extpsdk/build.gradle

```diff
@@ -10,6 +10,9 @@ ext {
     } else {
         pypi_repo = 'testpypi'
     }
+    if (!project.rootProject.hasProperty('vantiqConnectorSdkVersion')) {
```

Comment:
New version update + changes to allow for (but not require) command-line override of version to build